client/src/elm/Backend/ResilienceMessage/Model.elm

```diff
@@ -17,7 +17,8 @@ type alias ResilienceMessage =
 
 
 type ResilienceCategory
-    = ResilienceCategoryIntroduction
+    = ResilienceCategoryGuide
```

Comment:
Guide is not a category of message. Guide constant. Revert and use different approach.